docs/map-making/how-to-upgrade-legacy-map-features.md

```diff
@@ -1,35 +1,26 @@
 # How To Upgrade Legacy Map Features
 
-A list of (case sensitive) changes to upgrade legacy TripleA Game XML properties and options
-to the most recent.
+A list of changes to upgrade legacy TripleA Game XML properties and options to the most recent.
```

Comment:
As I said, here it is not clear whether the mentioned properties are the properties within the game file or the properties files of the map. Assuming that "TripleA Game XML properties and options" means the properties and options within the game (XML) file, this is not correct, as this "list of changes" is not only about that. That also would imply that this list of changes is useless for upgrading mapskins, while it actually contains information critical for such maps too.

Reply:
There is only one kind of XML file, it's clear that any XML examples are going to be for the game files. We are pretty explicit otherwise about the other files that need changing. We need to balance reasonable confusion with keeping this document concise. 

>  That also would imply that this list of changes is useless for upgrading mapskins

I don't necessarily draw that implication, but it could be worth clarifying perhaps. It probably is going to be obvious though that anyone upgrading a mapskin will skip over the XML parts.